tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist = {py3}-sklearn{0.23},flake8
+
+[flake8]
+max-line-length = 120
+
+[testenv]
+deps =
+    sklearn0.23: scikit-learn==0.23.1
```

Comment:
Should we also have the previous version here? Something like `sklearn 0.20.0: scikit-learn==0.20.0`.

Reply:
I tried adding this but running into an error as we have the sklearn installation even in requirements.txt